client/flutter/android/app/src/main/AndroidManifest.xml

```diff
@@ -33,7 +33,6 @@
               android:name="io.flutter.embedding.android.SplashScreenDrawable"
               android:resource="@drawable/launch_background"
               />
-            <meta-data android:name="firebase_analytics_collection_enabled" android:value="false" />
```

Comment:
This was changed because I had previously added this attribute in the wrong tag -- under `activity` -- rather than under `application`.